### PR TITLE
Support Python 2 *and* 3

### PR DESCRIPTION
**Pull request by Syeberman**

### CODE REVIEW

File: bearfield/encoders.py [hunk 1]
```diff
@@ -108,11 +110,11 @@ def encode(self, value):
         try:
             value = OrderedDict(value)
             encoded = OrderedDict()
-            for field, direction in value.iteritems():
+            for field, direction in six.iteritems(value):
                 direction = self.encode_int(field, direction)
                 field = self.encode_str(field, field)
                 encoded[field] = direction
-            return encoded.items()
+            return list(encoded.items())
```
Comment (wood-jp) [inline bearfield/encoders.py]: why cast to list?

File: tests/common.py [hunk 2]
```diff
@@ -24,7 +26,7 @@ def connection(self):
     @property
     def connections(self):
         """Return a dictionary of configured connections."""
-        return connection.connections.items()
+        return list(connection.connections.items())
```
Comment (wood-jp) [inline tests/common.py]: Same question, but also the comment here says the function will return a dictionary?

File: bearfield/encoders.py (hunk 1 above)
Comment (Syeberman) [inline bearfield/encoders.py]: In 2 it's a `list`, in 3 it's a "dictionary view" (iterable).  Downstream code might implicitly expect this to be a `list`, so we shouldn't change the type.

File: tests/common.py (hunk 2 above)
Comment (Syeberman) [inline tests/common.py]: Well....it _didn't_ return a dictionary.  I'll fix that comment.

Comment (Syeberman) [inline tests/common.py]: Actually, it wasn't used, so I've removed it.

**Pull request merged**